internal/postprocessor/main.go

```diff
@@ -311,18 +312,41 @@ func (p *postProcessor) getDirs() []string {
 
 func (p *postProcessor) MoveSnippets() error {
 	log.Println("moving snippets")
-	dirs := p.getDirs()
-	for _, dir := range dirs {
-
-		snpDirs, err := filepath.Glob(filepath.Join(dir, "apiv*", "internal"))
+	for _, module := range p.modules {
+		dir := filepath.Join(p.googleCloudDir, module)
+		snpDirs, err := filepath.Glob(filepath.Join(dir, "apiv*", "internal", "snippets"))
```

Comment:
I don't think this will work for things like spanner/admin/apiv1, right? If so I think the logic for toDir is slightly off too. Instead of iterating over modules consider p.config.GapicImportPaths(). The full client paths may be useful for proper placement in the snippets dir as well.

Reply:
Fixed, thanks!!!